packages/template/vite-typescript/tmpl/vite.preload.config.ts

```diff
@@ -1,23 +1,26 @@
 import type { ConfigEnv, UserConfig } from 'vite';
 import { defineConfig, mergeConfig } from 'vite';
-import { getBuildConfig, external, pluginHotRestart } from './vite.base.config';
+import { getBuildConfig, external, esmodule, pluginHotRestart } from './vite.base.config';
 
 // https://vitejs.dev/config
 export default defineConfig((env) => {
   const forgeEnv = env as ConfigEnv<'build'>;
   const { forgeConfigSelf } = forgeEnv;
+  const ext = esmodule ? 'mjs' : 'js';
```

Comment:
I think this always has to be `cjs` for preload scripts? Or at least it seems to need to be `cjs` when the package is type=module... i.e.,

```suggestion
  const ext = esmodule ? 'cjs' : 'js';
```

Reply:
https://github.com/electron/electron/blob/v30.0.0-nightly.20240104/docs/tutorial/esm.md#preload-scripts

Reply:
I saw that documentation, but in my local testing (with `package.json` type=module), the only working combination I could find for preload was `.cjs` compiled to common js, I believe with both isolatedModules and not.

I suspect that documentation may only be correct with an unset package.json type or type=commonjs?